search/serializers.py

```diff
@@ -303,6 +317,82 @@ class Meta:
         read_only_fields = ("text", "score", "created", "removed", "deleted")
 
 
+class ESBootcampSerializer(ESCourseSerializer):
+    """
+    Elasticsearch serializer class for bootcamps
+    """
+
+    object_type = BOOTCAMP_TYPE
+
+    class Meta:
+        model = Bootcamp
+        fields = [
+            "id",
+            "course_id",
+            "short_description",
+            "full_description",
+            "language",
+            "year",
+            "start_date",
+            "end_date",
+            "enrollment_start",
+            "enrollment_end",
+            "title",
+            "image_src",
+            "topics",
+            "prices",
+            "instructors",
+            "published",
+            "availability",
+        ]
+
+        read_only_fields = fields
+
+
+class ESProgramSerializer(ESModelSerializer):
+    """
+    Elasticsearch serializer class for programs
+    """
+
+    object_type = PROGRAM_TYPE
+
+    topics = serializers.SerializerMethodField()
+
+    def get_topics(self, program):
+        """
+        Get the topic names for a program
+        """
+        return list(program.topics.values_list("name", flat=True))
+
+    class Meta:
+        model = Program
+        fields = ["id", "short_description", "title", "image_src", "topics"]
+
+        read_only_fields = fields
+
+
```

Comment:
How do we want to search for Programs and LearningPaths? Currently, end users can only search for them by title, description, and topics. We can't search for courses in these Lists models. How we answer this question will affect `ESProgramSerializer` and `ESLearningPathSerializer`, which is currently only serializing what's shown.

Reply:
As mentioned above, only metadata, not objects within the lists.